src/book/02-system/03-tools/02-nubook.mdx

```diff
@@ -26,9 +26,15 @@ You'll need [Git](https://git-scm.com/), [Node.js](https://nodejs.org/en/) and [
    sudo apt install git
    ```
 
-2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) (pick the latest version).
+2. Install Node.js version 14 by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) and running the commands under "Node.js v14.x".
```

Comment:
I believe this should work with Node 16, will double check in the lab today. 16 is the current LTS so we should stick to that if possible

Reply:
Node 16 works too, I reckon we go with that

Reply:
Okay, I will go through the setup again and see if I can get the later versions to work.

Reply:
You're correct, Node 16 worked for me as well.